### ISSUE TITLE
Contributors?

### ISSUE THREAD

**Issue by richardliaw** [vesoft-inc/github-statistics#9]
Would it be possible show "Contributors" over time?

**Comment by Priceincoding** [comment]
It is absolutely possible as everything comes from the GithubAPI. Can you be specific on what kind of infomation will be helpful about "Contributors"?

**Comment by richardliaw** [comment]
Yes! I want to see, how popular is a project going over time. Specifically, what is the size of the set of contributors? 

One way of doing this right now on the CLI is `git shortlog -n -s -- . | wc -l`. It'd be great if that was part of the graphs.
